syntax = "proto3";

message AddedComment {
    string event_type = 1;
    int32 post_id = 2;
    string comment_by = 3;
    string comment =4;
    bool comment_status = 5;
  }

message DeletedComent {
  string event_type = 1; // "unliked"
  int32 post_id = 2;
//   string unliked_by = 3;
}

message EventWrapperComent {
  oneof event {
    AddedComment coment_posted = 1;
    DeletedComent deleted_coment = 2;
  }
}


